tools/corsavro_ft2ascii.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python3
+
+"""
+Simple tool to dump the contents of a corsaro flowtuple Avro file (i.e.
+one generated using corsaro3) to standard output.
+
+The format of the text output matches that of the old cors2ascii tool
+when run against a corsaro2 flowtuple file.
+
+The main purpose of this tool is to provide some backwards compatibility
+for people who were using cors2ascii with flowtuple data, and wish to
+continue to do so with the new avro files without having to re-develop any
+code / scripts.
+
+Usage: python3 corsavro_ft2ascii.py <flowtuple avro file>
+
+<flowtuple avro file> can be a swift URI.
```

Comment:
can it?

Reply:
Hah, wishful thinking on my part -- I did quickly try to make it work with pywandio, but the Avro reader object didn't like the bytes being provided to it by the wandio reader for some reason.